trezor-crypto/src/bip32.c

```diff
@@ -190,6 +199,41 @@ int hdnode_from_seed(const uint8_t *seed, int seed_len, const char* curve, HDNod
 	return 1;
 }
 
+int hdnode_from_seed_hd(const uint8_t *seed, int seed_len, const char* curve, HDNode *out)
```

Comment:
Could you also add a test for the Trezor part within Trezor?
make -Cbuild -j12 TrezorCryptoTests

Reply:
I just added the test!